cpp/bridge.cpp

```diff
@@ -257,7 +257,7 @@ BridgeResult opsqlite_execute_prepared_statement(
           // Intentionally left blank
 
         default:
-          row.values.push_back(JSVariant(NULL));
+          row.values.push_back(JSVariant(nullptr));
```

Comment:
I remember doing this and having some crashes, either on iOS or Android, did you test both?

Reply:
@ospfranco I've done tests here with Android 7, 10 and 11 devices, and Iphone 11 and 13 with IOS 17.2. For now, no crashes.
I belive that as NULL was a C macro that resolves 0, std::holds_alternative resolves TRUE on for <int>, returning ZERO in case of null value.

Reply:
Hmm I did write a test for this though, and I never got 0. Maybe you can check the test suite and add a test if missing?

Reply:
> Hmm I did write a test for this though, and I never got 0. Maybe you can check the test suite and add a test if missing?

Sure. I will do that.